pkg/controller/backup_sync_controller.go

```diff
@@ -122,7 +122,7 @@ func shouldSync(location *arkv1api.BackupStorageLocation, now time.Time, backupS
 }
 
 func (c *backupSyncController) run() {
-	c.logger.Info("Checking for backup storage locations to sync into cluster")
+	c.logger.Info("Waiting for existing backup storage locations to sync into cluster")
```

Comment:
I changed this message to be "Checking for existing backup storage locations...." which I think is more appropriate in all scenarios, including the one where BSLs **do** exist (also changed it to debug, so it's not filling up the log).  So I'd revert this change